CoeursVaillants/CoeursVaillants.html

```diff
@@ -7,8 +7,7 @@
         <label>Peuple <input type="text" name="attr_peuple"> Poids <input type="text" name="attr_poids"></label>
     </div>
     <div class="sheet-logo sheet-block">
-<!        <img class="sheet-logo" src="https://cdn.discordapp.com/attachments/896513056824234054/896513122121171004/unknown.png" width="100%" />!>
-        <img class="sheet-logo" src="https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/CoeursVaillants/images/logo.png" width="100%" />
+        <img class="sheet-logo" src="https://cdn.discordapp.com/attachments/896513056824234054/896513122121171004/unknown.png" width="100%" />
```

Comment:
You should have the images in the roll20 repo. Else they might be deleted and noone is around to fix it

Reply:
I push a new one, think this is good this time. >> Fix Logo (d61a397)

title="Version 1.00 - 10/12/2021" src="https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/CoeursVaillants/images/Logo.png"